crates/pop-contracts/README.md

```diff
@@ -115,4 +115,4 @@ let contract_address = call_smart_contract(call_exec,  weight_limit, url)
 ```
 
 ## Acknowledgements
-`pop-contracts` would not be possible without the awesome crate: [`cargo-contract`](https://github.com/paritytech/cargo-contract).
\ No newline at end of file
+`pop-contracts` would not be possible without the awesome crate: [`cargo-contract`](https://github.com/paritytech/cargo-contract).
```

Comment:
maybe "`this` awesome crate" as it's a singular mention?

Reply:
Not sure why it appears in this PR to be honest, it was a fix in another PR (A merge I guess).
Suggested by Daan here: https://github.com/r0gue-io/pop-cli/pull/107#discussion_r1562808571, let's keep it with `the`